.gitignore

```diff
@@ -72,3 +72,4 @@ typings/
 fragmentTypes.json
 
 data/
+eq-author-api/.firebase-admin-service-account.json
```

Comment:
I wouldn't bother with this to be honest, someone could name their key something different and still check it in.

Reply:
I requested this as if every dev needs it and it shouldn’t be checked in then every dev would need to set this up and the necessary docker-compose change. So it’s simpler to tell them to just set it up with that name and then it just works. Are you expecting a different workflow?

Reply:
Yeah I think it's a good idea, but adding the name seems quite specific, If we are going to add  a `.gitignore` rule I think a better approach would be to instruct developers to place their keys in a certain directory e.g. `/secrets` and to `.gitignore` the secrets directory instead.

Reply:
We could do but I can't imagine having many (if any) more. Seems fine to me but this is not a hill I'm willing to die on 😆

Reply:
Same here. Let's just go with this for now 👍